uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -280,7 +275,8 @@ private fun AppBar(
                         pagerState.animateScrollToPage(index)
                         tabLazyListStates.getValue(tab).animateScrollToItem(index = 0)
                     }
-                }
+                },
+                modifier = Modifier.zIndex(1f)
```

Comment:
I'm not sure, but the following comments may not be confusing. 👀 

```suggestion
                // To overwrite the indicator
                modifier = Modifier.zIndex(1f)
```